src/python/pants/backend/javascript/nodejs_project_environment.py

```diff
@@ -125,12 +125,6 @@ async def setup_nodejs_project_environment_process(req: NodeJsProjectEnvironment
         args = tuple(req.args)
     output_files = req.output_files
     output_directories = req.output_directories
```

Comment:
Why is the per_package_caches removed?

Reply:
I think I may have been thinking it was no longer necessary but I'm not sure why I thought that, and the removal here hardly seems comprehensive. Or maybe it was when trying to track down the bug related to corepack home. 
Anyway, put it back.